python/pyspark/sql/connect/session.py

```diff
@@ -625,6 +625,26 @@ def addArtifacts(self, *path: str, pyfile: bool = False, archive: bool = False)
 
     addArtifact = addArtifacts
 
+    def copyFromLocalToFs(self, local_path: str, dest_path: str) -> None:
```

Comment:
Can be move this to ML package? Should also mark it as a developer API in `Notes` section.

Reply:
I feel like this API is useful in many places, not only for ML, what about keeps putting it  here ? We can still mark it as developer API

Reply:
Can we add `.. versionadded 3.5.0` directive?